users/templates/user_notifications_list.html

```diff
@@ -1,36 +1,60 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
+{% load static %}
 {% load data_structure_element_selectors %}
 
+{% block extra_head %}
+    <link
+        rel="stylesheet"
+        href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css"
+        integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+        crossorigin="anonymous"
+    />
+    <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
+        crossorigin="anonymous"
+    />
+    <link
```

Comment:
This seems to be unnecessary because we are using font awesome in `base`

Reply:
@Szymiks is right, please extract this stylesheet to template from which both place inherit and don't repeat loading this script again.

Reply:
Done